data/human/culture conversations.txt

```diff
@@ -1071,6 +1071,150 @@ mission "Exotic Life: Geyser"
+			`	Then, she suddenly screams, "Only offworlders and blind fools with a death wish swim in that poison!"`
+			`	A wry grin floods her face, and she adds in a conversational tone, almost as an afterthought, "Now, I'll flay you alive and turn you into chum if you don't buy some fish bait. Eight racks." She slams a scarred hand into a rancid barrel beside her.`
+			choice
+				`	(Buy a barrel of rotten fish bait for 8,000 credits.)`
+					to display
+						credits > 8000
+				`	(Run for it.)`
+					goto declined
+			action
+				payment -8000
+			`	You grit your teeth against the stench and fight back the urge to gag as you fumble over the credits to the gnarled pirate. With an irritated snort, she shoves the putrid barrel towards you, and you quickly roll the it away. Once she's out of sight, you ditch the barrel and dart back to the <ship>, leaving behind a grim and grimy reminder of this lawless port.`
+				decline
+			label declined
+			`	You race through the maze of weathered crates and guarded stalls. As you push past the chaotic crowd, her bellowing laughter - a harsh, unbridled roar of contempt - echoes in your ears as you flee back to the safety of your <model>.`
+				decline
+
+phrase "old man waiting"
+	word
+		"Brooding in the dim light,"
+		"Sitting with a weary stare,"
+		"Resting his weathered gaze,"
+		"Hunched over silent memories,"
+		"Lost in quiet rumination,"
+	word
+		" "
+	word
+		"his fingers tap the scarred wood,"
+		"his eyes drift over the shadowed floor,"
+		"his breath mingles with the smoky air,"
+		"his silence is smothered by the conversations of bar patrons,"
+	word
+		" "
+	word
+		"awaiting your next question with a secretive look."
+		"with mystery etched into every line of his face."
+		"poised at the edge of revelation and reticence."
+		"anticipating the spark of your next question."
+
+mission "Exotic Life: Albatross"
+	minor
+	source Albatross
+	to offer
+		random < 10
+	on offer
+		conversation
+			`You trudge through the biting wind to a weathered spaceport bar, its exterior marked by the signs proclaiming "No Gods, No Masters," and "No Cybernetic Implants. No Badgers, Weasels, Or Polecats." Inside, the air is heavy with shadow and smoke - worn wood panels, dim, flickering lights, and cluttered signs declaring even more peculiar prohibitions. Slouched in the corner by the heavy door, an old man sits silently, his half-lidded eyes watching the drunken banter and clinking glasses. You could approach him, and perhaps press for answers about the bar's odd proclamations.`
+			label nextquestion
+			`	${old man waiting}`
+			choice
+				`	"Why no Cybernetic implants?"`
+					goto implants
+					to display
+						not "heardimplants"
+				`	"What's the deal with Badgers?"`
+					goto badgers
+					to display
+						not "heardbadgers"
+				`	"Turnips?"`
+					to display
+						has "heardbadgers"
+						not "heardturnips"
+					goto turnips
+				`	"The sign says 'No ${plural nouns}'?"`
```

Comment:
Not sure how to do a direct quote within the direct quote of the sign. Does this work if it only directly quotes the plural noun?
```suggestion
				`	"The sign says no '${plural nouns}'?"`
```